webui/src/pages/repositories/repository/branches.jsx

```diff
@@ -55,27 +56,28 @@ const BranchWidget = ({ repo, branch, onDelete }) => {
 
     return (
         <ListGroup.Item>
-            <div className="clearfix">
-                <div className="float-start">
-                    <h6>
+            <Row className="d-flex align-items-center justify-content-between">
+                <Col className="flex-grow-1">
+                    <h6 style={{marginBottom: 0}}>
                         <Link href={{
                             pathname: '/repositories/:repoId/objects',
                             params: {repoId: repo.id},
                             query: {ref: branch.id}
                         }}>
                             {branch.id}
                         </Link>
-
+                        
                         {isDefault &&
-                        <>
-                            {' '}
-                            <Badge variant="info">Default</Badge>
-                        </>}
+                            <>
+                                {' '}
+                                <Badge variant="info">Default</Badge>
+                            </>
+                        }     
                     </h6>
-                </div>
+                </Col>
 
 
-                <div className="float-end">
+                <Col md="3" className="d-flex justify-content-end">
```

Comment:
Does this look ok if you remove the `md="3"`, or is it required in this case?

Reply:
The `row` class has `.row > * { width: 100% }`, so something needs to have specific sizing if we want anything but a 50/50 split. It also has the added benefit of having responsiveness built in.